images/scala.svg

```diff
@@ -0,0 +1 @@
+<svg height="64" width="64" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><linearGradient id="a"><stop offset="0" stop-color="#656565"/><stop offset="1" stop-color="#010101"/></linearGradient><linearGradient id="b" gradientUnits="userSpaceOnUse" x1="13.528" x2="88.264" xlink:href="#a" y1="-36.176" y2="-36.176"/><linearGradient id="c" gradientUnits="userSpaceOnUse" x1="13.528" x2="88.264" xlink:href="#a" y1="3.91" y2="3.91"/><linearGradient id="d"><stop offset="0" stop-color="#9f1c20"/><stop offset="1" stop-color="#ed2224"/></linearGradient><linearGradient id="e" gradientUnits="userSpaceOnUse" x1="13.528" x2="88.264" xlink:href="#d" y1="-55.974" y2="-55.974"/><linearGradient id="f" gradientUnits="userSpaceOnUse" x1="13.528" x2="88.264" xlink:href="#d" y1="-15.87" y2="-15.87"/><linearGradient id="g" gradientUnits="userSpaceOnUse" x1="13.528" x2="88.264" xlink:href="#d" y1="24.22" y2="24.22"/><g transform="matrix(.492308 0 0 .492308 6.941539 39.876923)"><path d="m13.4-31s75 7.5 75 20v-30s0-12.5-75-20z" fill="url(#b)"/><path d="m13.4 9s75 7.5 75 20v-30s0-12.5-75-20z" fill="url(#c)"/><path d="m88.4-81v30s0 12.5-75 20v-30s75-7.5 75-20" fill="url(#e)"/><path d="m13.4-21s75-7.5 75-20v30s0 12.5-75 20z" fill="url(#f)"/><path d="m13.4 49v-30s75-7.5 75-20v30s0 12.5-75 20" fill="url(#g)"/></g></svg>
```

Comment:
there is a missing newline at the end of the file

Reply:
:heavy_check_mark: